src/runtime/runtime-test.js

```diff
@@ -467,19 +467,6 @@ describes.realWin('Runtime', {}, env => {
         }
       );
     });
-
```

Comment:
why remove these?

Reply:
I am removing Runtime's logEvent, so this test is no longer valid.